### PR TITLE
[WIP] Add performance tests for grpc service invocation

### PR DESCRIPTION
**Pull request by wmeints**
# Description

A new performance test for GRPC service invocation. This follows the same pattern as the performance test for HTTP service invocations.

## Issue reference

#1787 

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [ ] Created/updated tests

### CODE REVIEW

File: tests/apps/perf/service_invocation_grpc/app.go [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	}
+
+	if err := s.Start(); err != nil {
+		log.Fatalf("Server error: %v", err)
+	}
+}
+
+func runHandler(ctx context.Context, e *common.InvocationEvent) (out *common.Content, err error) {
+	out = &common.Content{
+		Data:        []byte("ok"),
```
Comment (yaron2) [inline tests/apps/perf/service_invocation_grpc/app.go:29]: No data should be sent back

**Review by yaron2 — CHANGES_REQUESTED** [review]

**Comment by yaron2** [comment]
@wmeints thanks for this PR. I have one comment for what's been pushed so far. we still need to have the test portion.

**Comment by wmeints** [comment]
Thanks for the feedback!

I've got the test portion on my computer. I'm trying to figure out how to invoke the fortio tool so it uses grpc. I ran the test on my machine and it fails with a disconnect. Will have more time over the weekend to take a look and fix my test code.

**Comment by yaron2** [comment]
> Thanks for the feedback!
> 
> I've got the test portion on my computer. I'm trying to figure out how to invoke the fortio tool so it uses grpc. I ran the test on my machine and it fails with a disconnect. Will have more time over the weekend to take a look and fix my test code.

Any updates? You could also write your own test app instead of Fortio if the latter has a problem.

**Comment by wmeints** [comment]
I'm actually working on the bits right now. I think I got beyond my initial problems. Will push an update when the test is working on my machine.

**Pull request closed without merging**